apps/xen-tg-app/src/App.tsx

```diff
@@ -11,20 +11,14 @@ import { TMemory } from './types'
 import { API_URL } from './env'
 
 const queryFn =
-  (
-    tgDataStr: string,
-    tgDataObj: WebAppInitData,
-    name: string,
-    params?: { [key: string]: string | number }
-  ) =>
-  async () => {
+  (tgDataStr: string, name: string, params?: { [key: string]: string | number }) => async () => {
```

Comment:
Remove the `tgDataStr` parameter here and below. Replace it with `window.Telegram.WebApp.initData` as you do in `mutationFn`.
`window.Telegram.WebApp.initData` is immutable and no sense to put it as `queryKey` to `useQuery` hook.